lib/Traits/Cesium3dTilesTraits.ts

```diff
@@ -0,0 +1,185 @@
+import { JsonObject } from "../Core/Json";
```

Comment:
I take it this file has just been renamed, which is fine, the github diff however isn't showing the deletion of `Cesium3dCatalogItemTraits`?

Reply:
Ignore this